assets/js/localization.js

```diff
@@ -207,17 +220,33 @@ function getLocales() {
         var localePairs = [];
         for(var code in locales) {
             localePairs.push([code, locales[code]]);
+            pageLocales[code] = locales[code];
```

Comment:
Um, `pageLocales` is an object storing `locale: corresponding language` as `key: value` pairs. I populate this object through AJAX call from `locales.json`. This was required because, for `select`, it was simple to just set the `val` to `locale` as the associated value held was the corresponding language. However, for the bootstrap button, to set its text, I fetch the locale and then set its text through this object. One more approach I had was to run a loop and fetch the language name from `locales[]`, but I chose this as it looked more cleaner and not much cluttering.